### DISCUSSION TITLE
Pause Socket的疑问

### DISCUSSION THREAD

**Discussion by ghost** [cloudwu/skynet#1813] [Q&A] [answered]
我有个mongo的服务，一直在报Pause Socket.
我看云大的blog描述
````
在网络方面，增强了流量控制的支持。这会对基于 skynet 做重 IO 操作（例如文件服务器）的业务有所帮助。在上个版本，对于写 socket 有 warning 消息的支持，可以让写缓冲区太大后，得到消息暂停业务层产生新数据；在写缓冲区清空后再延续
````
Pause Socket会暂停业务层产生新数据。这个业务层是指外面调用mongo服务的service吗，如果这样的话暂停了我外部调用mongo的service，是不是会导致我业务卡顿。

**Comment by cloudwu** [comment, marked as answer]
pause 表示已经收取的数据处理不完了，所以暂停从 socket 上接收。
